src/main/java/com/gregtechceu/gtceu/common/cover/FacadeCover.java

```diff
@@ -57,7 +57,7 @@ public boolean shouldRenderPlate() {
     /**
      * @return If the pipe this is placed on and a pipe on the other side should be able to connect
      */
-    public boolean blockPipePassThrough() {
+    public boolean canPipePassThrough() {
         return false;
```

Comment:
Not sure if the returned value also needs to change, as the renamed method has different semantics here.

Reply:
Nope, i also changed how it's used 
Also look at the javadoc, it defines the new semantics